packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue

```diff
@@ -34,31 +34,65 @@
         <span />
       </template>
     </oc-select>
-    <div class="oc-flex oc-flex-middle oc-flex-between oc-mb-l oc-mt-s">
+    <div class="oc-flex oc-flex-between oc-mb-l oc-mt-s">
       <role-dropdown
         :allow-share-permission="hasResharing || resourceIsSpace"
         mode="create"
         class="role-selection-dropdown"
         @option-change="collaboratorRoleChanged"
       />
-      <expiration-datepicker
-        v-if="!saving"
-        :share-types="selectedCollaborators.map((c) => c.value.shareType)"
-        @option-change="collaboratorExpiryChanged"
-      />
-      <oc-button
-        id="new-collaborators-form-create-button"
-        key="new-collaborator-save-button"
-        data-testid="new-collaborators-form-create-button"
-        :disabled="!$_isValid || saving"
-        :variation="saving ? 'passive' : 'primary'"
-        :appearance="saving ? 'outline' : 'filled'"
-        submit="submit"
-        :show-spinner="savingDelayed"
-        @click="share"
-      >
-        <span v-text="$gettext(saveButtonLabel)" />
-      </oc-button>
+      <div class="oc-flex">
+        <div v-if="formattedExpirationDate" class="oc-flex">
+          <oc-icon
+            v-oc-tooltip="formattedExpirationDate"
+            class="files-collaborators-collaborator-expiration"
+            data-testid="recipient-info-expiration-date"
+            :aria-label="formattedExpirationDate"
+            name="calendar-event"
+            fill-type="line"
+          />
+          <span class="oc-invisible-sr" v-text="screenreaderShareExpiration" />
+        </div>
+        <oc-button
+          id="show-more-actions-btn"
```

Comment:
I feel like this name is a bit too general for an ID that is supposed unique. You could simply add `oc-mx-s` as class and remove the ID and the CSS down below.

Reply:
I understand the concern, however, it is also used as dropdown toggle so we'd need an ID here.

Reply:
Ahh okay, true. Still, please use a less generic name then, something like `new-collaborators-form-more-actions-button` (since we already have `new-collaborators-form-create-button`).